test/integ/axpy/src/axpy.cpp

```diff
@@ -136,10 +136,12 @@ void operator()()
     Val * const pBufHostY = alpaka::mem::view::getPtrNative(memBufHostY);
 
     // C++11 random generator for uniformly distributed numbers in [0,1)
+    // keep in mind, this can generate different values on different platforms
     std::random_device rd{};
-    std::default_random_engine eng{ rd() };
+    unsigned long seed = rd();
```

Comment:
just use `auto const` here?
```suggestion
    auto const seed = rd();
```